schedule/kernel/ibtest-master.yaml

```diff
@@ -12,6 +12,7 @@ vars:
     IBTEST_IP1: 10.162.2.93
     IBTEST_IP2: 10.162.2.94
     IBTEST_ROLE: IBTEST_MASTER
+    IBTEST_TIMEOUT: 7200
```

Comment:
Is there a reason for this long timeout? The machine will be blocked for other jobs in this case, and a successful run should never even take an hour, does it?

Reply:
likely it wont. but i thought that this would be save. the 7200 exceeds the openQA timeout. So either it will run within the ibtest test or it will timeout from the openQA limits. in case that other tests added or the ibtest exceeds the 3200 (as before) it will not fail before the openqa tell it so.  i thought that this is save and i dont think has significant impact.

Reply:
The machines are shared with other jobs as well, so we should try to have as little waste of time as possible. Unless there is no need for a timeout of two hours, I would propose to stick with less.

Reply:
i removed the timeout and the results are https://openqa.suse.de/tests/8115897/logfile?filename=autoinst-log.txt. Adding back the timeout the test is able to finish https://openqa.suse.de/tests/8116064.

Reply:
https://openqa.suse.de/tests/8120524#settings (w/o ibtest_timeout and opnmpi3).
So now i am confused. But take this out of this PR. i will remove the /IBTEST_TIMEOUT/ and you can resolve the issue.

Reply:
*this request is ready for review*